### ISSUE TITLE
Create a new topic in Apache Pulsar

### ISSUE THREAD

**Issue by MPSebastian** [amosproj/amos2023ss01-apache-pulsar-ui#29]
## User story
1. As a user
2. I want to create new topics
3. So that i have new subjects to send/retireve data

## Acceptance criteria
* [ ] User can create new topic
* [ ] User gives a topic name as input

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
